File: Nível 1/2061 - As Abas de Péricles/main.py
valores = input("A: ").split(" ")

abas = int(valores[0])
quantia = int(valores[1])

for i in range(1, quantia + 1):
    texto = input("B: ")
    texto = texto.lower()

    if texto == "fechou":
        abas -= 1
        abas += 2
    
    if texto == "clicou":
        abas -= 1

print(abas)

File: Nível 1/2061 - As Abas de Péricles/main.lua
valores = io.read()
abas, quantia = valores:match("(%S+)%s+(%S+)")
abas = tonumber(abas)
quantia = tonumber(quantia)

for i = 1, quantia do
    texto = io.read()
    texto = string.lower(texto)

    if texto == "fechou" then
        abas = abas - 1
        abas = abas + 2
    end

    if texto == "clicou" then
        abas = abas - 1
    end
end

print(abas)